src/common/Network/PacketDef/Zone/ServerZoneDef.h

```diff
@@ -247,20 +247,20 @@ namespace Sapphire::Network::Packets::Server
     } entry[8];
   };
 
-  struct FFXIVIpcFreeCompanyMemberLogInOut : FFXIVIpcBasePacket< FreeCompanyMemberLogInOut >
+  struct FFXIVIpcFreeCompanyEvent : FFXIVIpcBasePacket< FreeCompanyEvent >
   {
     uint16_t unknown;
     uint16_t unknown1;
     uint16_t unknown2;
     uint16_t unknown3;
     uint16_t unknown4;
     char padding[6];
-    uint8_t status; // 0x0F Login, 0x10 Logout
+    uint8_t eventID; // 0x0F Login, 0x10 Logout
     uint8_t padding1;
     char padding2[6];
     uint16_t unknown5;
-    char fc[46];
-    char character[32];
+    char fcName[46];
+    char characterName[32];
```

Comment:
Based on testing they don't always seem to be fcName and characterName.
For example: `The company tag has been changed to "XXX"`. XXX uses what you put down as fcName.
So it's probably safe to assume they are parameters and don't have a set name.